symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/command/HelpCommand.java

```diff
@@ -20,14 +19,15 @@
  * A help command listing all the commands that can be performed by an end-user through the chat.
  */
 @API(status = API.Status.STABLE)
-public class HelpCommand extends PatternCommandActivity<CommandContext> {
+public class HelpCommand extends SlashCommand {
```

Comment:
don't we risk breaking the compatibility here? (not sure everyone would be extending the HelpCommand, maybe we can check on internal projects)

Reply:
Yes indeed. I discussed it with @symphony-thibault 
We assumed customers would either use SlashCommands through the static methods or or directly subclass PatterCommandActivity